classes/admin_portal_data.ts

```diff
@@ -0,0 +1,38 @@
+import firebase from '../firebase'
+
+export class AdminPortalData {
+  application_status: string // values are: 'removed', 'finalised for review' and 'finalised for interview'
+  review_marks: number
+  interview_marks: number
+
+  updateDetails(
+    application_status: string,
+    review_marks: number,
+    interview_marks: number,
```

Comment:
interview marks and review marks should be objects because interviewers will give category based marks. Those categories will work as keys and marks will be their values. The same goes for reviewers also.